### DISCUSSION TITLE
Could not resolve to subscribable node with the global id...

### DISCUSSION THREAD

**Discussion by gyund** [community/community#52128] [Mobile] [answered]
### Select Topic Area

Bug

### Body

When swiping on the GitHub app for android 1.107.0, in the notifications tab with multiple notifications, the message in the title above appears as a toast. The row remains visible but if you refresh the view, it will finally go away. Running pixel 7 android 13.

**Comment by stevepopovich** [comment]
Hi @gyund, is this still happening? Are you able to provide a screenshot with the toast? It would help us debug. Thanks!

**Reply by gyund** [reply]
![Screenshot_20230405-220647.png](https://user-images.githubusercontent.com/12544461/230918595-a2310ee0-b804-4818-8b27-ebff4c5b9615.png)

Here was one I took when it happened.

I was just trying it with some other notifications I had today for PRs and merges and they seemed to go away correctly. Still running the same version.

**Comment by stevepopovich** [comment, marked as answer]
Ok @gyund, here is what is going on. This type of notification is not able to be subscribed/unsubscribed too. This CI runs are a special 1 off type of notification. 

We need to do a better job of showing this to the user. We are thinking about how we can improve this, but know that _technically_ nothing is broken. Your notification should go away correctly and this error doesn't really mean anything to the user. Does that make sense?

**Reply by gyund** [reply]
Yep! It works as you describe. I just ignore the toast. 🍞 👍
